### PR TITLE
📊 hmd: iteration 2

### PR DESCRIPTION
**Pull request by lucasrodes**
- Update snapshot
- Add new metrics:
  - `life_expectancy_fm_ratio`: Life expectancy ratio female to male.
  - `life_expectancy_fm_diff`: Life expectancy difference, female minus male.
- Add variable descriptions.
- Add `deaths` table in meadow, integrate table in single main table in garden. (required for https://github.com/owid/etl/pull/1680)

### CODE REVIEW

File: etl/steps/data/garden/hmd/2023-09-19/hmd.meta.yml [hunk 1]
```diff
@@ -14,12 +25,13 @@ tables:
...
       central_death_rate:
         title: Central death rate
-        description_short: Central death rate between ages x and x+n.
-        unit: deaths per people
+        description_short: |-
+          For a given population or cohort, the central death rate at age x during a given period of 12 months is found by dividing the number of people who died during this period while aged x by the average number who were living in that age group during the period.
+        unit: deaths per 1,000 people
       probability_of_death:
         title: Probability of death
         unit: ""
```
Comment (spoonerf) [inline etl/steps/data/garden/hmd/2023-09-19/hmd.meta.yml:33]: Could multiply by 100 and present as a %, but up to you.

File: etl/steps/data/meadow/hmd/2023-09-19/hmd.py [hunk 2]
```diff
@@ -68,43 +77,70 @@ def run(dest_dir: str) -> None:
...
+    ## life exp
+    tb_lt.metadata = snap.to_table_metadata()
+    tb_lt.metadata.short_name = "life_tables"
+    for column in tb_lt.columns:
+        tb_lt[column].metadata.origins = tb_lt.metadata.dataset.origins
+    ## exposure to risk
+    tb_exp.metadata = snap.to_table_metadata()
+    tb_exp.metadata.short_name = "exposures"
+    for column in tb_exp.all_columns:
+        tb_exp[column].metadata.origins = tb_exp.metadata.dataset.origins
```
Comment (spoonerf) [inline etl/steps/data/meadow/hmd/2023-09-19/hmd.py:122]: Maybe this isn't needed any more after the recent origins update?

**Review by spoonerf — APPROVED** [review]
LGTM! v.minor suggestions

File: snapshots/hmd/2023-09-18/hmd.zip.dvc [hunk 3]
```diff
@@ -38,54 +26,49 @@ meta:
...
-    producer: Various sources
-    citation_full: HMD. Human Mortality Database. Max Planck Institute for Demographic
-      Research (Germany), University of California, Berkeley (USA), and French Institute
-      for Demographic Studies (France). Available at www.mortality.org.
+    producer: Human Mortality Database
+    citation_full: |-
+      HMD. Human Mortality Database. Max Planck Institute for Demographic Research (Germany), University of California, Berkeley (USA), and French Institute for Demographic Studies (France). Available at www.mortality.org.
+
+      See also the methods protocol:
+      Wilmoth, J. R., Andreev, K., Jdanov, D., Glei, D. A., Riffe, T., Boe, C., Bubenheim, M., Philipov, D., Shkolnikov, V., Vachon, P., Winant, C., & Barbieri, M. (2021). Methods protocol for the human mortality database (v6). [Available online](https://www.mortality.org/File/GetDocument/Public/Docs/MethodsProtocolV6.pdf).
```
[the remark this answers is not in the document]
Comment (lucasrodes) [reply snapshots/hmd/2023-09-18/hmd.zip.dvc:59]: Good catch. You need to log in to mortality.org. I'll add a note

File: etl/steps/data/garden/hmd/2023-09-19/hmd.meta.yml (hunk 1 above)
Comment (lucasrodes) [reply etl/steps/data/garden/hmd/2023-09-19/hmd.meta.yml:33]: Totally, I'll scale this

File: etl/steps/data/meadow/hmd/2023-09-19/hmd.py (hunk 2 above)
Comment (lucasrodes) [reply etl/steps/data/meadow/hmd/2023-09-19/hmd.py:122]: I am unsure how to use that at the moment ー Feels like I'd have to hardcode the origins fields again in Garden after having already defined them in Snasphot?

**Pull request merged**